tests/edit_inputs.sh

```diff
@@ -55,3 +56,46 @@ function edit_diag_table {
       -e "s/SMONTH/$SMONTH/g" \
       -e "s/SDAY/$SDAY/g"
 }
+
+function edit_ww3_input { 
+
+  SDATEWW3="${SYEAR}${SMONTH}${SDAY} $(printf "%02d" $(( ${SHOUR}  )))0000"
+  EDATEWW3="2100${SMONTH}${SDAY} $(printf "%02d" $(( ${SHOUR}  )))0000"
+  #note EDATE should be SDATE+FHMAX, but since this requires adding ndate 
+  #a work around is to just put a date long in the future as the actual end time is
```

Comment:
"Workaround" is one word, not two. Also, the "date" command can perform the same operations as the "ndate" command. Here's an overly-verbose example:

`/usr/bin/env TZ=UTC date -d "2021-09-14t18:30:00 UTC +12 hours"  +%Y-%m-%dt%H:%M:%S`
`2021-09-15t06:30:00`